dashboards/flow-data-for-circuits.json

```diff
@@ -382,7 +382,7 @@
       "type": "netsagenavigation"
     },
     {
-      "content": "<center><h1><b>Flow Data for Circuits</b></h1></center>\n<center>This dashboard provides flow data for the top ten sources and destinations by volume and rate for circuits.</center>\n<center>Click on an organization name in one of the tables to see the flow data dashboard for that organization.</center>\n<center><b>Please note that rate will be zero if only one sampled flow was detected.</b></center>\n<center>All times are displayed in browser local time.</center>\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>",
+      "content": "<center><h1><b>Flow Data for Circuits</b></h1></center>\n<center>This dashboard provides flow data for the top ten sources and destinations by volume and rate for circuits.</center>\n<center>Click on an organization name in one of the tables to see the flow data dashboard for that organization.</center>\n<center>All times are displayed in browser local time.</center>\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>",
```

Comment:
Looks like this reverted an earlier commit, please fix.

Reply:
Should be fixed now! :)